db/R/dbfiles.R

```diff
@@ -264,32 +286,39 @@ dbfile.posterior.insert <- function(filename, pft, mimetype, formatname, con, ho
 ##' \dontrun{
 ##'   dbfile.posterior.check(pft, 'application/x-RData', 'traits', dbcon)
 ##' }
-dbfile.posterior.check <- function(pft, mimetype, formatname, con, hostname=fqdn()) {
-  if (hostname == "localhost") hostname <- fqdn();
-
+dbfile.posterior.check <- function(pft, mimetype, formatname, con, hostname = fqdn()) {
+  if (hostname == "localhost") 
+    hostname <- fqdn()
+  
   # find appropriate pft
   pftid <- get.id("pfts", "name", pft, con)
   if (is.null(pftid)) {
-    invisible(data.frame())
+    return(invisible(data.frame()))
```

Comment:
Significant change: before an invisible `data.frame` was being created but not used, and execution continued. Now it's returned (as I thought was the intention).